validator/db/db.go

```diff
@@ -68,6 +69,8 @@ func NewKVStore(dirPath string, pubKeys [][48]byte) (*Store, error) {
 	if err := os.MkdirAll(dirPath, 0700); err != nil {
 		return nil, err
 	}
+	log.WithField("database-path", dirPath).Info("Checking DB")
+	log.WithField("validator-keys-path", cmd.DataDirFlag.Name).Info("Checking Validator Keys")
```

Comment:
Wouldn't this print `validator-keys-path=datadir`?

Reply:
By this i mean, you are printing the flag name, not the value of the flag.

Reply:
Oh sorry, What is the property I have to use to get the datadir?

Reply:
Hi @ShawkiS typically we can use something like `cliCtx.GlobalString(cmd.DataDirFlag.Name)`